src/clients/asset-client.js

```diff
@@ -113,12 +115,21 @@ class AssetClient {
 
   __createOrUpdate (options = {}) {
     const fileName = options.file.name
+    const variations = Object.keys(options.variations).map(key => {
+      return {
+        [key]: JSON.stringify(options.variations[key])
+      }
+    }).reduce((accumulator, object) => {
+      return Object.assign(accumulator, object)
+    }, {})
```

Comment:
Why do you have to do something like this?
I feel that could be optimized or removed.